src/DTC/Support/Can.ts

```
@@ -364,17 +389,12 @@ export class CanDecoder extends BaseDecoder {
   private _toHexString(value: number | null | undefined): string {
-    try {
-      if (value === null || value === undefined) return "null";
-      return "0x" + value.toString(16).padStart(2, "0").toUpperCase();
-    } catch {
-      return "invalid";
-    }
+    return value !== null && value !== undefined ? value.toString(16).toUpperCase() : "";
   }
 
   protected _log(level: LogLevel, ...message: unknown[]): void {
     if (false == false) {
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)